.github/DEVELOPERS.md

```diff
@@ -1,19 +1,24 @@
-The purpose of this document is to document the steps for a new developer to this project to get onboarded
-to be able to contribute to the project.
+The purpose of this document is to document the steps for a new developer of this project to get onboarded
+to be able to contribute.
 
 1 - The source code for this application is written in the Pascal programming language.
-Only Windows operating systems are currently supported as the target platform for the application to run.
+Only Windows operating systems are currently supported as the target platform for the application to operate on.
 
 1 - Fork the repo to your own GitHub account.
+git clone https://github.com/<yourAccountName>/MourseRunner.git
```

Comment:
<yourAccountName>/MourseRunner.git should be w7sst/MorseRunner.git

Reply:
I need some time to work out if what you stated would work do a fork and clone in one command.
I've realized that I neglected to state how to do a fork. It seemed like a no brainer to me, doing such on the GitHub GUI so many times previously on other projects.
So I'll be making a change, please don't merge this yet.

Reply:
Ahhh. Your instructions had assumed that you had created a fork (like you did under your user account). My change assumed you were cloning directly to your home machine. Yes, I think most people will create a fork.

Let me know when you want me to resume the review.